google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/features/ObjectApi.java

```diff
@@ -199,7 +198,7 @@ GoogleCloudStorageObject simpleUpload(@PathParam("bucket") String bucketName, @H
    @Named("Object:delete")
    @DELETE
    @Path("storage/v1/b/{bucket}/o/{object}")
-   @Fallback(TrueOnNotFoundOr404.class)
+   @Fallback(FalseOnNotFoundOr404.class)
```

Comment:
@snotling I opened [JCLOUDS-805](https://issues.apache.org/jira/browse/JCLOUDS-805) to track this issue.  I will take a quick look and asked @hsbhathiya, the original developer, to do the same.